templates/App/Item/_content.html.twig

```diff
@@ -33,6 +33,10 @@
                         <i class="fa fa-exchange fa-fw"></i>
                     </a>
                 {% endif %}
+                <a href="{{ path('app_item_generate_dublin_core_xml', {'id': item.id}) }}" class="button"
+                    title="{{ 'tooltip.generate_dublin_core_xml'|trans }}">
+                    <i class="fa fa fa-file-code-o fa-fw"></i>
+                </a>
```

Comment:
Should this option be available only for admin user? I don't see any reason to hide it from other users. Please consider

Reply:
No, it shouldn't be available only for admin.
Now it's available for regular user as well.